### PR TITLE
feat(ffi): support adding transports to provider

### PR DESCRIPTION
**Pull request by mefellows**

### CODE REVIEW

File: rust/pact_ffi/src/verifier/handle.rs [hunk 1]
```diff
@@ -65,6 +65,25 @@ impl VerifierHandle {
...
+  ) {
+
+    let transport = ProviderTransport {
+      transport: protocol,
+      port: Some(port),
+      path: if path.is_empty() { None } else { Some(path) },
+      scheme: scheme
+    };
+
+    self.provider.transports.append(&mut vec![transport]);
```
Comment (rholshausen) [inline rust/pact_ffi/src/verifier/handle.rs:84]: There must be a function to add a new item, as this one creates a new vector, converts it into a mutable slice, and then appends that to the vector

Comment (rholshausen) [reply rust/pact_ffi/src/verifier/handle.rs:84]: Yeah, `push` is what we want to use

**Comment by rholshausen** [comment]
This looks fine to me (apart from using append instead of push), so it must be an issue with the verifier function not using the transports

File: rust/pact_ffi/src/verifier/handle.rs (hunk 1 above)
Comment (mefellows) [reply rust/pact_ffi/src/verifier/handle.rs]: I _swear_ this was the first thing I tried, but it seems to compile now so 🤷 . Fixed, thanks.

**Pull request merged**